apps/aechannel/src/aesc_channels.erl

```diff
@@ -273,9 +274,9 @@ serialization_template(?CHANNEL_VSN) ->
     , {delegate_ids       , [id]}
     , {state_hash         , binary}
     , {round              , int}
+    , {solo_round         , int}
     , {lock_period        , int}
-    , {force_blocked_until, int}
-    , {closes_at          , int}
+    , {locked_until       , int}
```

Comment:
I see in line with https://github.com/aeternity/protocol/commit/1e4a362bf5aaca70b4390c0a2f308a1e8ccada47#diff-3976273847c0bfced542c67100a7990bR717 
ok